website/pages/en/sunrise.mdx

```diff
@@ -74,15 +74,15 @@ This new version will then sync in Subgraph Studio, a testing and sandbox enviro
 
 > Publishing requires Arbitrum ETH - upgrading your subgraph also airdrops a small amount to facilitate your first protocol interactions 🧑‍🚀
 
-### What happens if I don't upgrade my subgraph?
+### I use a subgraph developed by someone else, how can I make sure that my service isn't interrupted?
 
-Subgraphs will be queryable on the hosted service until the cutoff date, after which query endpoints will no longer be available, and owners will not be able to deploy new versions. Owners will still be able to simply upgrade their subgraphs to the network at any time, even after the cutoff date.
+When the owner has upgraded their subgraph, you will be able to easily go from the subgraph's hosted service page to the corresponding subgraph on The Graph Network, and update your application to use the new subgraph's query URL ([learn more](querying/querying-the-graph)).
 
-### I use a subgraph developed by someone else, how can I make sure that my service isn't interrupted?
+Around the start of June, Edge & Node will automatically upgrade actively queried subgraphs. This will give any third-party data consumers an opportunity to move subgraph endpoints to The Graph Network before June 12th. The subgraph owners will still be able to claim these subgraphs on the network using the hosted service upgrade flow.
 
-When the owner has upgraded their subgraph, you will be able to easily go from the subgraph's Hosted Service page to the corresponding subgraph on The Graph Network, and update your application to use the new subgraph's query URL ([learn more](querying/querying-the-graph)).
+### What happens if I don't upgrade my subgraph?
```

Comment:
This question will be updated to the following after June 12th.  (I will put in the PR request then) 

> I didn’t upgrade my subgraph by June 12th. What should I do if I still want to use it?
> 
> 
> The [hosted service](https://thegraph.com/hosted-service) homepage is still accessible and can be used to search for legacy hosted service subgraphs. If your hosted service subgraph has already been auto-upgraded, you may claim its network equivalent as the original owner. If your subgraph was not already auto-upgraded, you can still upgrade and publish it.